### ISSUE TITLE
Get `electronVersion` from `package.json` file

### ISSUE THREAD

**Issue by christian-bromann** [webdriverio-community/wdio-electron-service#150]
Addresses a comment made by an Electron core maintainer in https://github.com/electron/electron/pull/39380#discussion_r1287421204. Can we just pick the version from the `package.json`? 

cc @goosewobbler

**Comment by goosewobbler** [comment]
This should be straight forward though we should keep the override for the case where the tests are in a different repo from the application being tested.

**Comment by christian-bromann** [comment]
> we should keep the existing configuration item as an override

Yes, totally 👍

**Comment by goosewobbler** [comment]
@christian-bromann Is there a way to override the chromedriver version used by WDIO?  I got the service working again by removing all the launcher code (and the `wdio-chromedriver-service` dependency), but I wouldn't be surprised if there are issues around CD version matching.

**Comment by christian-bromann** [comment]
If you set `browserVersion` to version Electron requires Chromedriver to be, it should work. Since you set the `binary` in `goog:chromeOptions` it won't download any Chrome browser but will set up Chromedriver with the correct version.

**Comment by goosewobbler** [comment]
Ok, thanks, in that case I think we can keep the lookup for the Chromedriver version which matches a given electron version and pass the correct Chromedriver version string to WDIO for download.

**Comment by ForesightImaging** [comment]
In case this approach is of interest, we just bug out to the node_modules folder and grab the package.json for packages we care about. This is mainly for keeping packages like electron, electron-chromedriver and browserslist on the same page via a set of 'versionCheck' unit tests.

```
import fs from 'fs';
import path from 'path';

export function findPackageJson(startDir: string) {
    let cur = startDir;
    // eslint-disable-next-line no-constant-condition
    while (true) {
        try {
            const testPath = path.join(cur, 'package.json');
            fs.accessSync(testPath);
            fs.accessSync(path.join(cur, 'node_modules'));   // has to be the right one
            return testPath;
        } catch (e) {
            let next = path.normalize(cur + '../');
            if (next === cur) {
                throw new Error('package.json not found');
            }
            cur = next;
        }
    }
}

export function nodeModulesDir(inPackageJson: string) {
    return path.join(path.dirname(inPackageJson), 'node_modules');
}

export function packageVersion(inPackageName: string, inNodeModulesDir?: string): string {
    let node_modules = inNodeModulesDir || (nodeModulesDir(findPackageJson(__dirname)));
    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
    return (JSON.parse(fs.readFileSync(path.join(node_modules, inPackageName, 'package.json'), { encoding: 'utf8' })).version) as string;
}
```

**Comment by goosewobbler** [comment]
@ForesightImaging thanks, I'll be taking a look at this issue once the service works again with the latest WDIO versions - progress [here](#168).

**Comment by goosewobbler** [comment]
Released with v5.0.0
